fvtest/tril/tril/ast.hpp

```diff
@@ -24,7 +24,7 @@
 
 #include "ast.h"
 
-#include "OMR/TypeTraits.hpp"
+#include "include_core/OMR/TypeTraits.hpp"
```

Comment:
You shouldn't have to make this change. Are you running into include issues?

Reply:
Yes, it was saying `No such a file under this directory`. I am trying to change the include path and see what will happen.

Reply:
Try this: make the tril library link against `omr_base` in the [CMakeLists.txt file](https://github.com/eclipse/omr/blob/master/fvtest/tril/tril/CMakeLists.txt#L74)

Reply:
This does make the `include_core` directory visible to cmake. But it still cannot find files under the `compiler` directory. Do you know where is this `compiler` directory supposed to be included?